File: basics/Q76/main.go
package main

import (
	"fmt"
	"math"
)

func main() {
	var radian float64
	fmt.Print("Enter Radian: ")
	fmt.Scan(&radian)
	degree := radian * (180 / math.Pi)
	fmt.Println("Degree is", degree)
}

File: basics/Q76/main.rs
use std::io;

fn main() {
    let mut input = String::new();

    println!("Enter Radian:");
    io::stdin().read_line(&mut input).expect("Failed to read line");
    let radian: f64 = input.trim().parse().expect("Not a valid number");

    let degree = radian * (180.0 / std::f64::consts::PI);

    println!("Degree is {}", degree);
}